RecoDataProducts/inc/TriggerInfo.hh

```diff
@@ -2,34 +2,34 @@
 #define RecoDataProducts_TriggerInfo_hh
 
 #include "canvas/Persistency/Common/Ptr.h"
+#include "Offline/RecoDataProducts/inc/CaloCluster.hh"
+#include "Offline/RecoDataProducts/inc/KalSeed.hh"
+#include "Offline/RecoDataProducts/inc/HelixSeed.hh"
+#include "Offline/RecoDataProducts/inc/TimeCluster.hh"
+#include "Offline/RecoDataProducts/inc/CaloTrigSeed.hh"
+#include "Offline/RecoDataProducts/inc/CosmicTrackSeed.hh"
 #include <string>
 #include <vector>
 
 namespace mu2e {
 
-  class CaloCluster;
-  class KalSeed;
-  class HelixSeed;
-  class TimeCluster;
-  class CaloTrigSeed;
-  class CosmicTrackSeed;
   struct TriggerInfo
```

Comment:
The payload of TriggerInfo is now all deep copies.   Are you worried about the increase in payload and potential confusion of having multiple objects?